guide/05-working-with-the-spatially-enabled-dataframe/part3_data_io_writing_data.ipynb

```diff
@@ -0,0 +1,1816 @@
+{
```

Comment:
<p>The <code><a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html?highlight=from_table#arcgis.features.GeoAccessor.to_table" target="_blank">to_table()</a></code> method can be used to export data from a SeDF into non-spatial formats, such as <a href="https://en.wikipedia.org/wiki/Comma-separated_values" target="_blank">csv</a> files or <a href="https://pro.arcgis.com/en/pro-app/latest/help/data/geodatabases/overview/table-basics.htm" target="_blank">tables</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>